spmd/tensor/parallel/style.py

```diff
@@ -185,13 +209,13 @@ def make_output_tensor(
     Convert Output DTensor to a replicated DTensor first and then convert it to Tensor.
     Args:
         output (DTensor): output of module to be converted.
-        device_mesh (Optional[DeviceMesh]): :class:``DeviceMesh`` object needed to
+        device_mesh (Optional[DeviceMesh]): :class:`DeviceMesh` object needed to
             replicate the output and it needs to be a 1D ``device_mesh`` and we will
             throw exceptions if a non-1D ``device_mesh`` is passed in. If no
             ``device_mesh`` is passed in, we will reuse the one from output.
             Default: ``None``
     Return:
-        A :class:``torch.Tensor`` object converted from output DTensor.
+        A :class:`torch.Tensor` object converted from output DTensor.
```

Comment:
DTensor -> :class:`DTensor` maybe? Not strictly necessary however.

Reply:
DTensor part will be removed.